extensions/spotify-player/src/components/PlaylistItem.tsx

```diff
@@ -24,7 +26,7 @@ export default function PlaylistItem({ type, playlist }: PlaylistItemProps) {
       subtitle={subtitle}
       content={icon}
       accessories={[{ text: `${playlist?.tracks?.total} songs` }]}
-      actions={<PlaylistActionPanel title={title} playlist={playlist} />}
+      actions={actions}
```

Comment:
This should also work. I can change it if you like.
```suggestion
      actions
```